modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/api/SessionBuilderImpl.java

```diff
@@ -123,26 +125,21 @@ public SessionBuilder defaultPageSize(int pageSize) {
     /** {@inheritDoc} */
     @Override
     public SessionBuilder property(String name, @Nullable Object value) {
-        var prop = QueryProperty.byName(name);
-
-        if (prop != null) {
-            props.put(prop, value);
-        }
+        props.put(name, value);
```

Comment:
What reason to have that properties? It not use anywhere

Reply:
they are for future use

Reply:
they are for future use